documentation/docs/faq.md

```diff
@@ -1,6 +1,16 @@
+---
+image: /img/integration-services/logo/integration_services.png
+description: Integration Services Frequently Asked Questions.
+keywords:
+- FAQ
+---
+
 # FAQ
 
 ## When should I use the Integration Services instead of IOTA Identity and IOTA Steams?
-The Integration Services **abstracts** technical complexity of the Identity and Streams frameworks. Furthermore the usage is **independent of any technology** stack because the Integration Services can be accessed via REST APIs. All data is **cached** and can accessed without long waiting times. Additionally the Integration Services provide more functionality by allowing users to **search for channels** or validate channel data. Channels, channel subscriptions and verifiable credential can **easily be maintained**.
-## Can I use the Integration Services together with IOTA Stronghold?
 
+The Integration Services **abstracts** technical complexity of the Identity and Streams frameworks. Furthermore, the usage is **independent of any technology** stack because the Integration Services can be accessed via REST APIs. All data is **cached** and can be accessed without long waiting times. Additionally, the Integration Services provide more functionality by allowing users to **search for channels** or validate channel data. Channels, channel subscriptions and verifiable credential can **easily be maintained**.
```

Comment:
```suggestion
The Integration Services **abstracts** technical complexity of the Identity and Streams frameworks. Furthermore, the usage is **independent of any technology** stack because the Integration Services can be accessed via REST APIs. All data is **cached** and can be accessed without long waiting times. Additionally, the Integration Services provide more functionality by allowing users to **search for channels** or validate channel data. Channels, channel subscriptions, and verifiable credential can **easily be maintained**.
```